test/syntax-highlight.test.js

```diff
@@ -34,19 +34,19 @@ describe('syntax-highlight tests', () => {
   });
 
   test('code inside <pre><code>...</code></pre> should get marked up', () => {
-    // C++ code example, in regular text and as code
+    // C# code example, in regular text and as code
     const original = 'const int i = 5; <pre><code>const int i = 5;</code></pre>';
     const result = syntaxHighlighter(original);
     const expected =
-      'const int i = 5; <pre class="hljs cpp"><code><span class="hljs-keyword">const</span> <span class="hljs-keyword">int</span> i = <span class="hljs-number">5</span>;</code></pre>';
+      'const int i = 5; <pre class="hljs csharp"><code><span class="hljs-keyword">const</span> <span class="hljs-keyword">int</span> i = <span class="hljs-number">5</span>;</code></pre>';
```

Comment:
Should we be using `cLike` here instead of `csharp`?

Reply:
@c3ho the test calls [`highlightAuto`](https://github.com/Seneca-CDOT/telescope/blob/0e12e5291db3e9746de3924a1d5846db7cf1806e/src/backend/utils/html/syntax-highlight.js#L44) so when it runs [`highlightjs`](https://highlightjs.org/) (the library we're using for highlighting code) detects the piece of code entered for the test as `csharp`.